src/t_zset.c

```diff
@@ -2184,6 +2188,165 @@ inline static void zunionInterAggregate(double *target, double val, int aggregat
     }
 }
 
+int dictGetMaxElementLength(dict *d) {
```

Comment:
This method cannot be moved to dict.c because it assumes the element is an sds.
in fact, let's mark it, and a few other of the new implementation detail functions as `static`, this way their name and prefix doesn't matter as much.

but regardless, let's also try to improve the name.
the `dict` prefix is used for dict infrastructure, so it would be wrong here.
i see this file has 3 different prefixes: `zsl` for the skiplist related functions (some of which maybe should have been in a file of it's own), `zz` for ziplist encoding helpers, and `zset` for generic sorted set functions. so i guess `zset` prefix is right for all of these.

Reply:
Done, just added the zset prefix, I tried some other variations, but I think just the prefix was better.